oidc-ui/configure_start.sh

```
@@ -0,0 +1,101 @@
+
+echo "Replacing public url placeholder with public url"
+
+workingDir=$nginx_dir/html
+if [ -z "$OIDC_UI_PUBLIC_URL" ]; then
+  rpCmd="s/_PUBLIC_URL_//g"
+  grep -rl '_PUBLIC_URL_' "$workingDir" | while IFS= read -r file; do
+    sed "$rpCmd" "$file" > "${file}.tmp" && mv "${file}.tmp" "$file"
+  done
+else
+  workingDir=$nginx_dir/${OIDC_UI_PUBLIC_URL}
+  mkdir -p $workingDir
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)